### ISSUE TITLE
Send meeting invite through google calendar

### ISSUE THREAD

**Issue by ramezrafla** [alextselegidis/easyappointments#333]
Right now the webapp sends an email with the meeting info. If we are using Google Calendar, we could use that to send meeting requests (it has a higher chance to pass spam filters and can get added to user calendar on outlook etc.). It seems Google Calendar does not send the invite for some reason.

I looked at the PHP code for google sync and adding apts. Could you guide me on what to add? I'll happily do it and PR. Thanks

**Comment by alextselegidis** [comment]
Hi @ramezrafla, do you mean the ICS files? 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by ramezrafla** [comment]
Yes - the standard email confirmation from google calendar that contains ics

**Comment by alextselegidis** [comment]
Ok I see, no worries I will include this in the upcoming relase :) 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by a-Tell** [comment]
Is it maybe possible to make this optional? I use my private calendar for Event-Scheduling, but I don't want to send Invites from it to my customers.

**Comment by alextselegidis** [comment]
Yeah, we could make this an option I guess. 

I will revise this before the actual implementation to make sure we get things right. 

<small>
<table>
<tr>
<td><img src="https://easyappointments.org/img/easyappointments-signature.png" style="margin-right: 5px"></td>
<td style="line-height: 2">Alex Tselegidis, Easy!Appointments Creator <br><a href="https://easyappointments.org/get-a-free-quote">Need a customization? Get a free quote!</a></td>
</tr>
</table>
</small>
